ide/projectui/src/org/netbeans/modules/project/ui/OpenProjectList.java

```diff
@@ -595,7 +606,7 @@ public boolean isDone() {
         @Override
         public Project[] get(long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, TimeoutException {
             long ms = unit.convert(timeout, TimeUnit.MILLISECONDS);
-            if (!TASK.waitFinished(timeout)) {
+            if (!waitFinished(timeout)) {
```

Comment:
This is the important change. Unit tests that call `OpenProjectList.getDefault().get()` could wait indefinitely long. Now they trigger the actual loading mechanism by calling `waitFinished` and not `TASK.waitFinished`. As a result it is more comfortable to write tests that invoke code relying on the list of open projects being loaded.